File: vscode/syntax_comparison/04-hello-module.clj
(ns hello-world)

(def greeting
  "Hello")

(defn greet [name]
  (println greeting name))

(def fn-test
  (greet "Logan"))

(defmacro ignore [expression]
  (print "Printing what I want instead"))

(def macro-test
  (ignore (print "This text won't be printed")))

(if foo
  (println "Bar")
  (+ this that))

File: vscode/syntax_comparison/04-hello-module.rb
class HelloWorld
  @greeting = "Hello"

  def greet(name)
    print @greeting + name
  end

  @fnText = greet("Logan")

  def ignore(expression)
    print "Printing what I want instead"
  end

  @macroTest = ignore(print("This text won't be printed"))
end
